boskos/janitor/janitor.go

```diff
@@ -61,11 +68,11 @@ func setup(c boskosClient, janitorCount int, bufferSize int) chan string {
 	return buffer
 }
 
-func run(c boskosClient, buffer chan string) int {
+func run(c boskosClient, buffer chan string, res string) int {
 	totalAcquire := 0
 
 	for {
-		if proj, err := c.Acquire("project", "dirty", "cleaning"); err != nil {
+		if proj, err := c.Acquire(res, "dirty", "cleaning"); err != nil {
```

Comment:
say if later on we want to add more types of projects - large-project, aws-project, it can be easier to deploy another janitor pod rather than change actual janitor code.